text/000x-lowering-deposits-assethub.md

```diff
@@ -0,0 +1,197 @@
+| collectionDeposit         | 0.1 KSM                          | 0.006688 KSM              |
+| itemDeposit               | 0.001 KSM                        | 0.000167 KSM              |
+| metadataDepositBase       | 0.006709666617 KSM               | 0.0006709666617 KSM       |
+| attributeDepositBase      | 0.00666666666 KSM                | 0.000666666666 KSM        |
+
+
+### Enhanced Approach to Further Lower Barriers for Entry
+In an effort to further lower barriers to entry and foster greater inclusivity, we propose additional modifications to the pricing structure. These proposed reductions are based on a collaborative and calculated approach, involving the consensus of leading NFT creators within the Polkadot and Kusama Asset Hub communities. The adjustments to deposit amounts are not made arbitrarily. Instead, they are the result of detailed discussions and analyses conducted with prominent NFT creators. 
+
+**Proposed Code Adjustments**
+```rust
+parameter_types! {
+	pub const NftsCollectionDeposit: Balance = deposit(1, 130);
+	pub const NftsItemDeposit: Balance = deposit(1, 164) / 40;
+	pub const NftsMetadataDepositBase: Balance = deposit(1, 129) / 10;
+	pub const NftsAttributeDepositBase: Balance = deposit(1, 0) / 10;
+	pub const NftsDepositPerByte: Balance = deposit(0, 1);
+}
+```
+
+**Prices and Proposed Prices on Polkadot Asset Hub:**
+
+**Polkadot**
+| **Name**                  | **Current price implementation** | **Proposed Prices** |
+|---------------------------|----------------------------------|---------------------|
+| collectionDeposit         | 10 DOT                           | 0.20064 DOT         |
+| itemDeposit               | 0.01 DOT                         | 0.005 DOT           |
+| metadataDepositBase       | 0.20129 DOT                      | 0.002 DOT           |
+| attributeDepositBase      | 0.2 DOT                          | 0.002 DOT           |
+
+**Kusama**
+| **Name**                  | **Current price implementation** | **Proposed Price in KSM** |
+|---------------------------|----------------------------------|---------------------------|
+| collectionDeposit         | 0.1 KSM                          | 0.006688 KSM              |
+| itemDeposit               | 0.001 KSM                        | 0.000167 KSM              |
+| metadataDepositBase       | 0.006709666617 KSM               | 0.0006709666617 KSM       |
+| attributeDepositBase      | 0.00666666666 KSM                | 0.000666666666 KSM        |
+
+
+## Discussion of Other Proposals
+Several innovative proposals have been considered to enhance the network's adaptability and manage deposit requirements more effectively:
+
+### Enhanced Weak Governance Origin Model
+The concept of a weak governance origin, controlled by a consortium like the System Collective, has been proposed. This model would allow for dynamic adjustments of NFT deposit requirements in response to market conditions, adhering to storage deposit norms. 
```

Comment:
What is the System Collective?

Reply:
https://forum.polkadot.network/t/polkadot-assethub-high-nft-collection-deposit/4262/16